### PR TITLE
Update ROADMAP.md

### PR DESCRIPTION
**Pull request by bunnyhero**
Throw a rope to those of us actually using the Swift SDK

## Checklist

- [x] I've read the [Contributing Guidelines](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] I've completed the [Contributor License Agreement](https://developers.facebook.com/opensource/cla)
- [x] I've ensured that all existing tests pass and added tests (when/where necessary)
- [x] I've ensured that my code lints properly: (`swiftlint && swiftlint autocorrect --format`)
- [x] I've updated the documentation (when/where necessary) and [Changelog](CHANGELOG.md) (when/where necessary)
- [x] I've added the proper label to this pull request (e.g. `bug` for bug fixes)

## Pull Request Details

Describe what you accomplished in this pull request

## Test Plan

Test Plan: **Add your test plan here**

### CODE REVIEW

**Comment by ajfigueroa** [comment]
Would this be better documented as an issue instead of in the README?

File: ROADMAP.md [hunk 1]
```diff
@@ -111,6 +111,9 @@ A contrived example but the same strategy will work for implementing more advanc
 #### Q: Can I continue to use the Objective-C SDK if I'm a Swift Developer?
 - Yes but over time you will see warnings and ultimately deprecations of methods called from the Objective-C SDK.
 
+#### Q: I was using the [Swift SDK](https://github.com/facebook/facebook-swift-sdk), but now nothing compiles
```
Comment (joesus) [inline ROADMAP.md]: This is completely valid point and it is as good a place as any to address it. I think we could benefit from updating some of the wording and suggested remedies.

```suggestion
#### Q: In the recent past there have been unexpected breaking changes to this project. Why? 

- Between v0.6.0 and v0.7.0 - see: [comparison](https://github.com/facebook/facebook-swift-sdk/compare/v0.6.0...v0.7.0) - we updated to point to version 5.0 of the ObjC SDK. This was a breaking change and we should have cut a major release to reflect that.

Those changes constituted a major effort to enhance the usability of the Swift interface that's generated from the ObjC SDK but it was not communicated well in terms of this project. The changes are documented reasonably well in this changelog under the [5.0 release](https://github.com/facebook/facebook-objc-sdk/blob/master/CHANGELOG.md#500) of the [Objective-C SDK](https://github.com/facebook/facebook-objc-sdk). There will be no future updates to major versions of the Objective-C SDK without corresponding updates to the Swift SDK. This caused a lot of churn and we'd like to provide a better experience than that.
```

File: ROADMAP.md [hunk 2]
```diff
@@ -111,6 +111,9 @@ A contrived example but the same strategy will work for implementing more advanc
...
+- You might be able to get some guidance on what changes you'll have to make based on [this commit](https://github.com/facebook/facebook-swift-sdk/commit/335ad2c0dd0d277759046a1e4b0d87643ca176e1), which updates the sample apps so they compile. Unfortunately there is no further documentation of the changes at this time.
```
Comment (joesus) [inline ROADMAP.md]:
```suggestion
```

**Comment by joesus** [comment]
> Would this be better documented as an issue instead of in the README?

I think it's valid to have this linked to the README. There are a number of open issues that I've been going through that are traced to this same cause. This might prevent future issues from being opened since people can find the breaking changeset more easily.

**Pull request merged**